.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain
+# consistent coding styles between different editors and IDEs.
+#
+# Grab a plugin for your editor of choice here http://editorconfig.org/#download
+
+root=true                           # most-top config file in .editorconfig configuration tree
```

Comment:
Please use not more than 2 spaces after commented line if you commenting inline.